### ISSUE TITLE
Cannot find Slack installation

### ISSUE THREAD

**Issue by gauss256** [fsavje/math-with-slack#20]
It stopped rendering math so I tried to reinstall. I get the error message "Cannot find Slack installation".

Windows 64-bit
Slack version: 2.7.1 64-bit Beta Channel

**Comment by fsavje** [comment]
The script is not made to automatically find beta versions of slack, so you'll need to specify the installation yourself.

There will be a folder something like "AppData\Local\slack\2.7.1-beta\resources\app.asar.unpacked\src\static" in your user profile folder, call the script with that as a provided parameter. E.g.:

```
math_with_slack.bat %gauss256%\AppData\Local\slack\%2.7.1-beta%\resources\app.asar.unpacked\src\static
```

where you need to change %gauss256% and %2.7.1-beta% to the appropriate values.

Please let me know if this work.

**Comment by gauss256** [comment]
OK, I've figured it out. For some reason the installation .bat file stopped working. Fixing it required the use of delayed expansion.

I'll do a pull request.

**Comment by gauss256** [comment]
@fsavje Our comments crossed. Fortunately, finding the beta version was not a problem. (How did I get on the beta channel anyway??)

Not only is the installation working again in my fork, but the math is rendering properly too.

**Comment by fsavje** [comment]
This should be addressed with version 0.2.2. Please let me know if it isn't.

**Comment by gauss256** [comment]
Works here.
